build/buildrelease.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$1" = "" ]]
+then
+  echo "Usage: buildrelease.sh version"
+  echo "e.g. buildrelease.sh 2.0.0"
+  echo "or buildrelease.sh 2.0.0-rc1"
+  echo "It is expected that a git tag will already exist"
+  exit 1
+fi
+
+declare -r VERSION=$1
+declare -r SUFFIX=$(echo $VERSION | cut -s -d- -f2)
+declare -r BUILD_FLAG=$([ "$SUFFIX" != "" ] && echo "--version-suffix $SUFFIX")
+declare -r OUTPUT=artifacts
+
+rm -rf releasebuild
+git clone https://github.com/nodatime/nodatime.git releasebuild
+cd releasebuild
+git checkout $VERSION
```

Comment:
```sh
git checkout "${VERSION}"
```
just to make sure that we pass a single argument to `checkout`.

(This if the first use of `${VERSION}`, and so will hopefully fail if we've passed a version containing a space because it won't exist.)